### DISCUSSION TITLE
Does the Final 'qCBF' File Include Partial Volume Correction?

### DISCUSSION THREAD

**Discussion by suflorcita** [ExploreASL/ExploreASL#1818] [Q&A]
Hello,

I have been running the pipeline on several images, but I am unsure whether the final "qCBF" file in the Population directory is corrected for partial volume effects. In the "dataPar.json" file, I enabled the option bPVCNativeSpace. Could you confirm if this applies the correction?

Thanks!

**Comment by MDijsselhof** [comment]
Dear Suflorcita,

Thank you for using ExploreASL! Could you provide us with some details about the ASL sequence you are using and the goal of your study? That way, we can provide a more detailed answer and possibly give advice. 

Regarding your question, the qCBF files in the population directory are not partial volume corrected, the correction is performed in the population module to correct ROI-based CBF values. These files can be found in the ExploreASL/Population/Stats folder and contain PVC0 (no correction) or PVC2 (partial volume corrected) within the name. Using 'bPVCNativeSpace' should indeed provide partial volume corrected CBF maps in native space, so in the subject/ASL_1 folder, they are named 'CBF_GM' for grey matter partial volume corrections for example. I hope this helps!

Mathijs, on behalf of the ExploreASL team

**Comment by suflorcita** [comment]
Dear Mathijs and the ExploreASL team,

Thank you very much for the response and clarification!

We’re using a Siemens 3D GRASE PASL sequence, and our study aims to conduct a cross-sectional analysis comparing two populations.  I wanted to ask if there is any transform available in the folder for converting the partial volume corrected (PVC) maps from native space to standard space.

Thank you once again for your help and insights!

Best regards,
Sol

**Reply by HenkMutsaerts** [reply]
Hi Sol,

This is the y_ASL.nii file in the ASL session/run folder (our derivatives are not yet in BIDS format).
You can use the function `xASL_spm_deformations` for this.

BTW: ExploreASL by default performs PVC on ROI level, after the Population module has run, you should find in the stats folder files with PVC0 (no PVC) and PVC2 (classical PVC but on ROI level). This should be more robust than doing PVC on the voxel-wise level as you are doing.

Hope this helps!

Best,


Henk

**Reply by MDijsselhof** [reply]
Dear Sol,

I see @HenkMutsaerts is helping you with the additional steps.  Would you be so kind as to fill in the [questionnaire](https://docs.google.com/forms/d/e/1FAIpQLSfOweardrFPtNzTwDPAisySBuz45nF6-MPogvnioAwO8-0yig/viewform), so we can keep track of the requirements of our users? 

Best wishes,

Mathijs
